packages/shared/src/graphql/notifications.ts

```diff
@@ -22,6 +22,19 @@ export interface NotificationAttachment {
 
 export enum NotificationType {
   System = 'system',
+  CommunityPicksFailed = 'community_picks_failed',
```

Comment:
Do we use it somewhere?

Reply:
In the events, we can let go of this whole enum and just turn it into a `string`. But if in case, I just wanted them to be complete for anyone to use.

Reply:
I don't think it's needed on the client side at this point. In the future if needed we can add it again